blackduck/HubRestApi.py

```diff
@@ -845,7 +845,7 @@ def get_or_create_project_version(self, project_name, version_name, parameters =
         if project:
             version = self.get_version_by_name(project, version_name)
             if not version:
-                self.create_project_version(project, version_name, parameters)
+                self.create_project_version(project, version_name, parameters=parameters)
                 version = self.get_version_by_name(project, version_name)
         else:
             self.create_project(project_name, version_name, parameters)
```

Comment:
should this also be changed? parameters=parameters?

Reply:
Yes, that is correct